README.md

```diff
@@ -50,16 +50,20 @@ Go to [Releases](https://github.com/MonitorControl/MonitorControl/releases) and
 
 | MonitorControl version | macOS version     |
 | ---------------------- | ----------------- |
-| v4.0.0                 | Big Sur 10.15*     |
+| v4.0.0                 | Catalina 10.15*   |
```

Comment:
Can't believe I messed this up lol especially since I had noticed it myself but somehow changed it back to the wrong name.